ui.apps/src/main/content/jcr_root/apps/wcm/core/content/editor/.content.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="nt:unstructured"
+        jcr:title="AEM Sites Page Editor"
+        sling:resourceType="cq/gui/components/authoring/editors/core"
+        consoleId="cq-sites-editor-page"
+        mainClientLib="cq.authoring.editor.sites.page"
+        onboardingPath="/mnt/overlay/wcm/core/content/editor/onboarding.html"
+        trackingFeature="Page Editor">
+        <sidepanels/>
+        <content jcr:primaryType="nt:unstructured">
+            <items/>
+        </content>
+    </jcr:content>
+</jcr:root>
```

Comment:
Should we be providing the content for these overalys? Dont we want to let the product to provide all this?

Reply:
When you say product, do you refer to AEM product code or did you mean the client code?

Reply:
@davidjgonzalez how do we  create a request to the product to add the above overlay? Or if long process, is it acceptable to document it in the component documentation, for clients to add?